framework/src/modules/dpos_v2/commands/pom.ts

```diff
@@ -92,32 +99,41 @@ export class ReportDelegateMisbehaviorCommand extends BaseCommand {
 	// eslint-disable-next-line @typescript-eslint/no-empty-function
 	public async execute(context: CommandExecuteContext<PomTransactionParams>): Promise<void> {
 		const { chainID, getMethodContext, params, transaction, header } = context;
+
 		const currentHeight = header.height;
 		const header1 = BlockHeader.fromBytes(params.header1);
 		const header2 = BlockHeader.fromBytes(params.header2);
+
+		header1.validate();
+		header2.validate();
```

Comment:
The headers have already been validated in command's `verify()` method.

What's the reason to re-validate them here? 🤔

Reply:
Updated the functionality.